src/sonar-scanner-executable.js

```diff
@@ -101,13 +102,23 @@ function getSonarScannerExecutable(passExecutableCallback) {
   var downloadUrl = baseUrl + fileName
   log(`Downloading from ${downloadUrl}`)
   log(`(executable will be saved in cache folder: ${installFolder})`)
-  download(downloadUrl, installFolder, { extract: true })
-    .on('response', res => {
-      bar.total = res.headers['content-length']
-      res.on('data', data => bar.tick(data.length))
-    })
+  var downloader = new DownloaderHelper(downloadUrl, installFolder)
```

Comment:
Use const or let instead, sonar smells

Reply:
I'd noticed that, but since the rest of the code uses `var` I ended up following the same style.
I'm gonna fix it.

Reply:
Done!